### ISSUE TITLE
Softmax applied twice

### ISSUE THREAD

**Issue by betogaona7** [betogaona7/melanomAI#1]
Softmax is applied in the last layer, and also in the loss function. The formula for cross entropy loss is _softmax() + NLLoss()_ that affects the network behavior. Should remove the softmax in the last layer.
